CONTRIBUTING.md

```diff
@@ -1,28 +1,100 @@
 # Contributing to the AllenSDK
 
-Thank you for your interest in contributing!
+Thank you for your interest in contributing! There are a few ways you can contribute
+to the AllenSDK project:
 
-### Bug reports and feature requests
+* Answering questions from users asking for help on the [forums](https://community.brain-map.org/tag/allensdk) or on [StackOverflow](https://stackoverflow.com/questions/tagged/allen-sdk)
+* Adding test coverage
+* Updating release documentation
+* Reporting bugs
+* Requesting features
+* Contributing code
 
-* Before reporting a bug or requesting a feature, use Github's issue search to see if anyone else has already done so.
-* If there is no existing issue, create a new one. You should include:
-    * A brief, descriptive title
-    * A clear description of the problem or request.
-    * If you are reporting a bug, your description should contain the following information:
-        * What you did (preferably the actual code or commands that you ran)
-        * What happened
-        * What you were expecting to happen
-        * How your system is configured (operating system, Python version)
-* If you are comfortable addressing this issue yourself, take a look at the [guide to contributing code](#code) below.
+## Table of Contents
+* [Answering User Questions](##Answering-UserQuestions)
+* [Bug Reports and Feature Requests](##Bug-Reports-and-Feature-Requests)
+    * [Reporting Bugs](###Reporting-Bugs)
+    * [Suggesting Features/Enhancements](###Suggesting-Features/Enhancements)
+    * [Asking Questions](###Asking-Questions)
+* [Contributing Code](##Contributing-Code)
+    * [Deciding What to Contribute](###Deciding-What-to-Contribute)
+    * [Setting Up](###Setting-Up)
+    * [Style Guidelines](###Style-Guidelines)
+        * [Docstrings](####Docstrings)
+    * [Testing Guidelines](###Testing-Guidelines)
+    * [Committing Guidelines](###Committing-Guidelines)
+    * [Making a Pull Request](###Making-a-pull-request)
+    * [Review Process](###Review-process)
 
-### Questions 
+
+## Answering User Questions
+A great way to contribute to the AllenSDK is to help answer user questions on 
+the [forums](https://community.brain-map.org/tag/allensdk) or [StackOverflow](https://stackoverflow.com/questions/tagged/allen-sdk). Assisting new users is not only a valuable service for the
```

Comment:
very minor, but you might want to link to the general community forum rather than the allensdk tag. Rationale:
1. questions that involve the allensdk don't always explicitly mention it (e.g. "How do I access these data?")
2. I suspect that people who are reading these guidelines have enough neuroscience knowledge to be useful on non-allensdk questions

Reply:
Thanks, I'll make this change